airflow/www/templates/airflow/chart.html

```diff
@@ -39,4 +64,9 @@
       </div>
     </div>
 
+    <script src="{{ url_for('static', filename='d3.v3.min.js') }}"></script>
+      <script src="{{ admin_static.url(
```

Comment:
I think these two crept include (@68-69 and @70) aren't needed anymore as I moved it to the parent template in a recent PR.